### PR TITLE
Improved root generation times.

### PR DESCRIPTION
**Pull request by CommandrMoose**
Readds the logic to generate blocks in the TARDIS dimension.

Optimizations have been done to reduce the load on the server when doing it.

### CODE REVIEW

File: common/src/main/java/whocraft/tardis_refined/common/tardis/TardisArchitectureHandler.java [hunk 1]
```diff
@@ -70,7 +77,7 @@ public static void buildAirlockEntranceFromStructure(StructureTemplate template,
                     Optional<StructureTemplate> structureNBT = level.getLevel().getStructureManager().get(new ResourceLocation(TardisRefined.MODID, "corridors/airlock_entrance"));
                     structureNBT.ifPresent(structure -> {
                         BlockPos offsetPosition = new BlockPos(3, 2, 0);
-                        structure.placeInWorld(level.getLevel(), pos.subtract(offsetPosition), pos.subtract(offsetPosition), new StructurePlaceSettings(), level.getLevel().random, 3);
+                        structure.placeInWorld(level.getLevel(), pos.subtract(offsetPosition), pos.subtract(offsetPosition), new StructurePlaceSettings(), level.getLevel().random, 2);
```
Comment (Jeryn99) [inline common/src/main/java/whocraft/tardis_refined/common/tardis/TardisArchitectureHandler.java:80]: The "2" should be the flag field from the Block class, we are trying NOT to have magic numbers

File: common/src/main/resources/data/tardis_refined/dimension_type/tardis.json [hunk 2]
```diff
@@ -5,8 +5,8 @@
   "respawn_anchor_works": true,
   "bed_works": true,
   "has_raids": false,
-  "has_skylight": false,
-  "has_ceiling": false,
+  "has_skylight": true,
```
Comment (Jeryn99) [inline common/src/main/resources/data/tardis_refined/dimension_type/tardis.json]: What is the benefit of a sky light?

Comment (CommandrMoose) [reply common/src/main/resources/data/tardis_refined/dimension_type/tardis.json]: Remnant from experimentation. I've reverted it.

File: common/src/main/java/whocraft/tardis_refined/common/tardis/TardisArchitectureHandler.java (hunk 1 above)
Comment (CommandrMoose) [reply common/src/main/java/whocraft/tardis_refined/common/tardis/TardisArchitectureHandler.java:80]: Updated

**Pull request merged**